### ISSUE TITLE
Flogger 0.7 JAR in Maven Central is missing classes

### ISSUE THREAD

**Issue by rcd47** [google/flogger#297]
Flogger 0.7 from Maven Central blows up during initialization with an error that ends with:
```
Caused by: java.lang.IllegalStateException: No logging platforms found:
com.google.common.flogger.backend.system.DefaultPlatform: java.lang.NoClassDefFoundError: com/google/common/flogger/util/Checks
	at com.google.common.flogger.backend.Platform$LazyHolder.loadFirstAvailablePlatform(Platform.java:98)
	at com.google.common.flogger.backend.Platform$LazyHolder.<clinit>(Platform.java:66)
	... 11 more
```

It appears that `com.google.common.flogger.util.Checks`, and possibly some others in the same package, are missing from the JAR.

**Comment by cgdecker** [comment]
Thanks for the report... looks like this was caused by some changes to the BUILD files that (unbeknownst to me) affected what files were deployed to Maven Central. I'll get a fixed release out tomorrow.

**Comment by cgdecker** [comment]
I've released [0.7.1](https://github.com/google/flogger/releases/tag/flogger-0.7.1) which should have the missing classes included. I'm planning on trying to make sure we run tests against the jar that will get uploaded to Maven in the future.
